flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/vector/VectorizedColumnBatch.java

```diff
@@ -132,4 +133,28 @@ public Decimal getDecimal(int rowId, int colId, int precision, int scale) {
 			return Decimal.fromUnscaledBytes(precision, scale, bytes);
 		}
 	}
+
+	public SqlTimestamp getTimestamp(int rowId, int colId, int precision) {
```

Comment:
Can you add a `TimestampColumnVector` too? Because the orc needs to have its own implementation.

Reply:
OK